sgeop/simplify.py

```diff
@@ -265,12 +266,20 @@ def simplify_singletons(
         new["_status"] = "new"
         new.geometry = new.simplify(max_segment_length * simplification_factor)
         new_roads = pd.concat([cleaned_roads, new], ignore_index=True)
+        agg = {"_status": _status}
+        for c in cleaned_roads.columns.drop(cleaned_roads.active_geometry_name):
+            if c != "_status":
```

Comment:
@martinfleis this issue was is these chunks where we are adding the `agg` dict, the initial `{"_status": _status}` was being overwritten with `{"_status": 'first'}`. That was wreaking havoc on the results.